<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-PO3T4V3" failures="6" tests="31" name="testCases.LoginPopUpTest" time="250.141" errors="0" timestamp="2022-07-11T10:29:03 AMT">
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLabelsPresenceVerification_Test" time="0.691"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogoPresenceVerification_Test" time="0.045"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpCloseButtonFunctionality_Test" time="0.714"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpEyeButtonFunctionality_Test" time="0.312"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpRememberMeLabelClickAbility_Test" time="0.268"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpCheckBoxClickAbility_Test" time="0.241"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpForgotPasswordLink_Test" time="0.146"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpSignUpLink_Test" time="0.264"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInPositiveTest" time="1.633"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.363"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="1.744">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testCases.LoginPopUpTest.loginPopUpLogInNegativeTest(LoginPopUpTest.java:199)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="1.566">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testCases.LoginPopUpTest.loginPopUpLogInNegativeTest(LoginPopUpTest.java:199)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.337"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.409"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="1.593">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testCases.LoginPopUpTest.loginPopUpLogInNegativeTest(LoginPopUpTest.java:199)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="1.638">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testCases.LoginPopUpTest.loginPopUpLogInNegativeTest(LoginPopUpTest.java:199)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="1.622">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testCases.LoginPopUpTest.loginPopUpLogInNegativeTest(LoginPopUpTest.java:199)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- loginPopUpLogInNegativeTest -->
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="30.444"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.459"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpErrorMessageValidationInvalidUsername" time="1.363"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpErrorMessageValidationInvalidPassword" time="1.376"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpLabelsPresenceVerification_Test" time="0.219"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpLogoPresenceVerification_Test" time="0.025"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpSupportLink_Test" time="0.328">
    <failure type="java.lang.AssertionError" message="did not expect [https://craftbet.com/forgot-password] but found [https://craftbet.com/forgot-password]">
      <![CDATA[java.lang.AssertionError: did not expect [https://craftbet.com/forgot-password] but found [https://craftbet.com/forgot-password]
at testCases.PasswordRecoveryTest.RecoveryPopUpSupportLink_Test(PasswordRecoveryTest.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- RecoveryPopUpSupportLink_Test -->
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpPositiveTest" time="0.459"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="0.188"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="0.222"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="0.216"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="0.201"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="0.206"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="0.263"/>
  <testcase classname="testCases.BaseTest" name="@AfterSuite finish" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite finish -->
</testsuite> <!-- testCases.LoginPopUpTest -->
